.github/workflows/linux.yml

```diff
@@ -1363,6 +1363,8 @@ jobs:
       INSTALL_DIR: ${{ github.workspace }}/install
       INSTALL_TEST_DIR: ${{ github.workspace }}/install/tests
       MODEL_HUB_TESTS_INSTALL_DIR: ${{ github.workspace }}/install/tests/model_hub_tests
+    if: fromJSON(needs.smart_ci.outputs.affected_components).TF_FE.test ||
+      fromJSON(needs.smart_ci.outputs.affected_components).TFL_FE.test
```

Comment:
does this step also performs benchmarking of TFL models?

Reply:
It helps us to trigger these perf tests in case of changes in any component affecting TF FE